tests/clients/test_ngsi_v2_timeseries.py

```diff
@@ -31,120 +32,165 @@ def setUp(self) -> None:
-                print(entity.to_pandas())
-
-            attr_values_type = client.get_entity_attr_values_by_type(
-                entity_type=self.entity_1.type, attr_name="temperature")
-            for entity in attr_values_type:
-                print(entity.to_pandas())
+            for entity in entities:
+                # get by id
+                attrs_id = client.get_entity_by_id(entity_id=entities[0].id,
+                                                   aggr_period='minute',
+                                                   aggr_method='avg',
+                                                   attrs='temperature,co2')
+                logger.info(attrs_id.json(indent=2))
+                logger.info(attrs_id.to_pandas())
+
+                attrs_values_id = client.get_entity_values_by_id(
+                    entity_id=entity.id)
+                logger.info(attrs_values_id.to_pandas())
+
+                attr_id = client.get_entity_attr_by_id(
+                    entity_id=entity.id, attr_name="temperature")
+                logger.info(attr_id.to_pandas())
+
+                attr_values_id = client.get_entity_attr_values_by_id(
+                    entity_id=entity.id, attr_name="temperature")
+                logger.info(attr_values_id.to_pandas())
+
+                # get by type
+                attrs_type = client.get_entity_by_type(
+                    entity_type=entity.type)
+                for entity_id in attrs_type:
+                    logger.info(entity_id.to_pandas())
+
+                attrs_values_type = client.get_entity_values_by_type(
+                     entity_type=entity.type)
+                for entity_id in attrs_values_type:
+                    logger.info(entity_id.to_pandas())
+
+                attr_type = client.get_entity_attr_by_type(
+                    entity_type=entity.type, attr_name="temperature")
+                for entity_id in attr_type:
+                    logger.info(entity_id.to_pandas())
+
+                attr_values_type = client.get_entity_attr_values_by_type(
+                    entity_type=entity.type, attr_name="temperature")
+                for entity_id in attr_values_type:
+                    logger.info(entity_id.to_pandas())
 
     def tearDown(self) -> None:
         """
         Clean up server
         Returns:
             None
         """
-        try:
-            for sub in self.cb_client.get_subscription_list():
-                self.cb_client.delete_subscription(sub.id)
-        except:
-            pass
-        try:
-            self.cb_client.delete_entity(entity_id=self.entity_1.id)
-            self.cb_client.delete_entity(entity_id=self.entity_2.id)
-        except:
-            pass
-
-        self.client.close()
+        clear_all(fiware_header=self.fiware_header,
+                  cb_url=settings.CB_URL,
+                  ql_url=settings.QL_URL)
+
+        self.ql_client.close()
         self.cb_client.close()
```

Comment:
Is this as well as each other teardown still needed? If the test fails is the header clean_up then not executed? And even if, each test cleans the state before its execution. 
In fact it probably does not harm us if we also execute a clean in the teardown, but I think we could skip it?

Reply:
@RedingJeff currently, if the test fails an error is raised. Hence, everything from there will not be executed. I can check if we can catch and reraise, but I think that is what teardown is for. It is basically a backup.

Reply:
Then we should simply leave it like this. It has no negative effect, better to have a backup than code that is 0.5 seconds faster in testing, but fails due to state issues.